pkg/handlers/webhookreceiver.go

```diff
@@ -284,9 +285,52 @@ func (o *ocmsdkclient) SendServiceLog(n *oav1alpha1.Notification, firing bool) e
 
 	req.Bytes(slAsBytes)
 
-	_, err = req.Send()
+	res, err := req.Send()
+	if err != nil {
+		return err
+	}
 
-	return err
+	err = responseChecker(res)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// responseChecker checks the ocm response returns error or not
+func responseChecker(r *sdk.Response) error {
+	opId := r.Header("X-Operation-ID")
+
+	if r.Status() == 201 {
+		log.WithField(LogFieldPostServiceLogOpId, opId).Info("service log sent succeeded")
+		return nil
+	}
+
+	var ocmRes OCMResponseBody
+	err := json.Unmarshal(r.Bytes(), &ocmRes)
+	if err != nil {
+		return err
+	}
+
+	log.WithFields(log.Fields{LogFieldPostServiceLogOpId: opId, LogFieldPostServiceLogFailedReason: ocmRes.Reason}).Error("service log sent failed")
+
+	switch r.Status() {
+	case 400:
+		return fmt.Errorf("validation errors occurred")
+	case 401:
+		return fmt.Errorf("invalid auth token")
+	case 403:
+		return fmt.Errorf("unauthorized to perform operation")
+	case 500:
+		return fmt.Errorf("internal server error")
+	default:
+		return fmt.Errorf("unknown Service Log return code")
```

Comment:
To make debugging potential ocm response errors, would it make sense to include the `r.Status()` value?